src/org/daisy/dotify/formatter/impl/obfl/ObflParserImpl.java

```diff
@@ -1579,6 +1582,37 @@ private void parsePageNumber(
         fc.insertPageReference(refId, style);
     }
 
+    private void parseExternalReference(
+            BlockContentBuilder fc,
+            XMLEvent event,
+            XMLEventIterator input
+    ) throws XMLStreamException {
+        Iterator<Attribute> it = event.asStartElement().getAttributes();
+        Map<QName, String> attrList = new HashMap<>();
+        while (it.hasNext()) {
+            Attribute a = it.next();
+            MetaDataItem newItem = new MetaDataItem(
+                new QName(null, a.getName().getPrefix(), "xmlns"),
```

Comment:
Note that the URI normally associated with "xmlns" is "http://www.w3.org/2000/xmlns/". Not so important what we use in this case, but just for your information.

Reply:
Not important but added it for completeness.